deploy/deployment_dev_cluster.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: registration-service-v0.1
+  namespace: toolchain-registration-service
+spec:
+  lookupPolicy:
+    local: false
+  tags:
+  - from:
+      kind: DockerImage
+      name: registry.svc.ci.openshift.org/codeready-toolchain/registration-service-v0.1:registration-service
+    importPolicy:
+      scheduled: true
+    name: registration-service
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    app: registration-service
+  name: registration-service
+  namespace: toolchain-registration-service
+spec:
+  replicas: 3
```

Comment:
how does this 3 number come?

Reply:
Done in 8af6a1af6a8d69f5871dffae62436a7dd8b011f2